<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FixtureCalibration</class>
 <widget class="QDialog" name="FixtureCalibration">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function Wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="m_introTab">
      <attribute name="title">
       <string>Introduction</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QTextBrowser" name="m_introText">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:600;&quot;&gt;Welcome to the QLC+ wizard!&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;This is a guided procedure that will allow you to start using QLC+ in a few minutes.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;It basically consists in three simple steps:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;1- add fixtures&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;2- select capabilities to create functions&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;3- add widgets to the Virtual Console&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;To move from a step to another press the &amp;quot;Next&amp;quot; button&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTextBrowser" name="m_wizardLogo">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="html">
          <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/wizard_256.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_fixturesTab">
      <attribute name="title">
       <string>Fixtures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="5">
        <widget class="QTextBrowser" name="m_fxHint">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/wizard.png&quot; width=&quot;24&quot; /&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt; Add the fixtures for which you want to create functions and widgets&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>211</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_functionsTab">
      <attribute name="title">
       <string>Functions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0">
          <widget class="QTreeWidget" name="m_fixtureTree">
           <property name="toolTip">
            <string>Fixtures that will be included in automatic function creation</string>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Fixtures</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QSlider" name="sliderTilt">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSlider" name="SliderPan">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QSpinBox" name="spinBoxTilt">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="2" column="8">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="grpPosition">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <widget class="QWidget" name="verticalLayoutWidget">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>30</y>
                <width>160</width>
                <height>121</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QRadioButton" name="rbLeftF">
                 <property name="text">
                  <string>Position Left Front</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbRightF">
                 <property name="text">
                  <string>Position Right Front</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbLeftB">
                 <property name="text">
                  <string>Position Left Back</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbRightB">
                 <property name="text">
                  <string>Position Right Back</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbCenter">
                 <property name="text">
                  <string>Position Center</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QPushButton" name="m_restrictButton">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>260</y>
                <width>121</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Restrict</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TILT</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>PAN</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBoxPan">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>120</number>
           </property>
           <item>
            <widget class="QToolButton" name="m_addButton">
             <property name="toolTip">
              <string>Select fixtures that will be included in the automatically created functions</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="qlcui.qrc">
               <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_removeButton">
             <property name="toolTip">
              <string>Don't include selected fixtures in the created functions</string>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset resource="qlcui.qrc">
               <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="6" colspan="2">
          <layout class="QGridLayout" name="m_gridLayoutArea">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
          </layout>
         </item>
         <item row="1" column="6" colspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Calibration</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="lblNomFixture">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>150</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="m_applyallButton">
           <property name="text">
            <string>Apply All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_vcTab">
      <attribute name="title">
       <string>Virtual Console</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>55</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/wizard.png&quot; width=&quot;24&quot; /&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt; Based on the functions you selected, I can create the following widgets on your Virtual Console. Just check which ones you need.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="m_widgetsTree">
         <column>
          <property name="text">
           <string>Widgets</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="m_nextButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="qlcui.qrc">
         <normaloff>:/forward.png</normaloff>:/forward.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="qlcui.qrc">
         <normaloff>:/check.png</normaloff>:/check.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="qlcui.qrc">
         <normaloff>:/uncheck.png</normaloff>:/uncheck.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qlcui.qrc"/>
  <include location="qlcui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_okButton</sender>
   <signal>clicked()</signal>
   <receiver>FixtureCalibration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FixtureCalibration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addButton</sender>
   <signal>clicked()</signal>
   <receiver>FixtureCalibration</receiver>
   <slot>slotAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAddClicked()</slot>
  <slot>slotRemoveClicked()</slot>
 </slots>
</ui>
